google/cloud/bigtable/tests/data_integration_test.cc

```diff
@@ -587,3 +587,50 @@ TEST_F(DataIntegrationTest, TableSampleRowKeysTest) {
   auto last = samples.back();
   EXPECT_LT(0, last.offset_bytes);
 }
+
+TEST_F(DataIntegrationTest, TableReadMultipleCellsBigValue) {
+  if (UsingCloudBigtableEmulator()) {
+    // TODO(#151) - remove workarounds for emulator bug(s).
+    return;
+  }
+
+  std::string const table_id = RandomTableId();
+  auto table = CreateTable(table_id, table_config);
+
+  std::string const row_key = "row-key-1";
+  // cell vector contains 8 cell of ~32 MiB
+  auto const MiB = 1024L * 1024L;
+
+  std::string value((32 * MiB) - 1024, 'a');
+  std::vector<bigtable::Cell> created;
+  std::vector<bigtable::Cell> expected;
+
+  std::string col_qualifier;
+  for (int i = 0; i < 8; i++) {
+    col_qualifier = "c" + std::to_string(i);
+    created.push_back(
+        bigtable::Cell(row_key, family, col_qualifier, 0, value, {}));
+    expected.push_back(
+        bigtable::Cell(row_key, family, col_qualifier, 0, value, {}));
+  }
+
+  CreateCells(*table, created);
+  auto result = table->ReadRow(row_key, bigtable::Filter::PassAllFilter());
+
+  if (not result.first) {
+    return;
+  }
+  int totalrowsize = 0;
+  for (auto& cell : result.second.cells()) {
+    totalrowsize += cell.value().size();
+  }
+  EXPECT_LE(totalrowsize, (256 * MiB));
```

Comment:
Shouldn't this be BIGTABLE_CLIENT_DEFAULT_MAX_MESSAGE_LENGTH instead of `256 * MiB`?

Reply:
BIGTABLE_CLIENT_DEFAULT_MAX_MESSAGE_LENGTH is define inside client_options.cc
should i move it out to the .h or where should i define it so it could be accessibly to data_integration_tests?

Reply:
Please leave `BIGTABLE_CLIENT_DEFAULT_MAX_MESSAGE_LENGTH` in the `.cc` file. We can live with a bit of duplication in tests (at some level we want duplication, the point is to check that the default good enough for this large value).